crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -1257,10 +1263,9 @@ pub fn export_registry_types(In(params): In<Option<Value>>, world: &World) -> Br
                     .iter()
                     .any(|c| schema.reflect_types.iter().any(|cc| c.eq(cc)))
             {
-                return false;
+                return None;
             }
-
-            true
+            Some((id.to_string(), schema))
         })
         .collect::<HashMap<String, JsonSchemaBevyType>>();
```

Comment:
Nit: Unfortunate we have to allocate (`to_string`) here since we made `export_types` return `Cow`. Would converting this to `HashMap<Cow<'static, str>, JsonSchemaBevyType>` be too difficult/involved here? If so, we can address this another time.

Reply:
I would love to put it into the next PR and have this one merged 😅